src/app/components/ChannelInfo/index.tsx

```diff
@@ -0,0 +1,196 @@
+import moment from 'moment';
+import { connect } from 'react-redux';
+import { Icon, Button, Modal } from 'antd';
+import Identicon from 'components/Identicon';
+import Unit from 'components/Unit';
+import { CHANNEL_STATUS } from 'lib/lnd-http';
+import { AppState } from 'store/reducers';
+import { getAccountInfo } from 'modules/account/actions';
+import { closeChannel } from 'modules/channels/actions';
+import { ChannelWithNode } from 'modules/channels/types';
+import { channelStatusText } from 'utils/constants';
+import { ellipsisSandwich, enumToClassName } from 'utils/formatters';
+import './style.less';
+
+interface StateProps {
+  account: AppState['account']['account'];
+}
+
+interface DispatchProps {
+  getAccountInfo: typeof getAccountInfo;
+  closeChannel: typeof closeChannel;
+}
+
+interface OwnProps {
+  channel: ChannelWithNode;
+  close?(): void;
+}
+
+type Props = StateProps & DispatchProps & OwnProps;
+
+interface DetailRow {
+  label: string;
+  value: React.ReactNode;
+}
+
+class ChannelInfo extends React.Component<Props> {
+  componentDidMount() {
+    if (!this.props.account) {
+      this.props.getAccountInfo();
+    }
+  }
+
+  render() {
+    const { account, channel } = this.props;
+    if (!account) {
+      return null;
+    }
+
+    let statusClass = `is-${enumToClassName(channel.status)}`;
+    if (channel.status === CHANNEL_STATUS.OPEN) {
+      statusClass = `${statusClass} is-${channel.active ? 'active' : 'inactive'}`;
+    }
+    
+    return (
+      <div className="ChannelInfo">
+        <div className="ChannelInfo-parties">
+          <div className="ChannelInfo-parties-party is-from">
+            <div className="ChannelInfo-parties-party-icon">
+              <Identicon pubkey={account.pubKey} />
+            </div>
+            <div className="ChannelInfo-parties-party-name">
+              {account.alias || account.pubKey}
+            </div>
+          </div>
+          <div className="ChannelInfo-parties-arrow">
+            <Icon type="swap" />
+          </div>
+          <div className="ChannelInfo-parties-party is-to">
+            <div className="ChannelInfo-parties-party-icon">
+              <Identicon pubkey={channel.node.pub_key} />
+              <div className={`ChannelInfo-parties-party-icon-status ${statusClass}`} />
+            </div>
+            <div className="ChannelInfo-parties-party-name">
+              {channel.node.alias || channel.node.pub_key}
+            </div>
+          </div>
+        </div>
+
+        <table className="ChannelInfo-details">
+          <tbody>
+            {this.getChannelDetails().map(d => (
+              <tr className="TxInfo-details-row" key={d.label}>
+                <td className="TxInfo-details-row-label">{d.label}</td>
+                <td className="TxInfo-details-row-value">{d.value}</td>
+              </tr>
+            ))}
+          </tbody>
+        </table>
+
+        {/* TODO: show button when LND issue #2730 is resolved */}
```

Comment:
Thanks for leaving this in, should be simple to hook up once this is fixed or we figure out way to switch to gRPC.